compute/admin_guide/vulnerability_management/agentless_scanning.adoc

```diff
@@ -21,6 +23,27 @@ By default, Prisma Cloud uses the default security group for connections.
 If needed, you can specify a custom security group in the configuration settings.
 A default VPC is also required for the scan to complete. 
 
+endif::compute_edition[]
+
+ifdef::prisma_cloud[]
+
+=== Prerequisites
+
+* https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/connect-your-cloud-platform-to-prisma-cloud/onboard-your-aws-account/add-aws-cloud-account-to-prisma-cloud.html[Onboard your AWS account], and ensure you select the *Monitor and Protect* mode on Prisma Cloud. 
++
+The specific permissions to perform Agentless scanning in your AWS cloud accounts are documented in the CFT for the *Read-Write (Limited)* role.
+
+* For currently onboarded accounts, if onboarded using the *Monitor* mode, switch to *Monitor and Protect* or selectively update the permissions for agentless scanning from the permissions documented in the template for the *Read-Write (Limited)* role. 
+
+* If the existing account was onboarded fpr *Monitor and Protect*,update the CFT for the new permissions.
+
+* Ensure you have connectivity to the Prisma Cloud administrative console from your cloud account.
++
+Prisma Cloud uses the default VPC and security group.
+For agentless scanning, Prisma Cloud creates temporary EC2 instances and infrastructure within this VPC for the scan and uses the security group to communicate with the Prisma Cloud console.
```

Comment:
Prisma Cloud uses the default VPC and security group. You can change the security group used in the scan configuration settings. 

You can remove the temporary EC2 instance line, too much detail.